src/yuzu/configuration/configure_general.h

```diff
@@ -34,6 +38,9 @@ class ConfigureGeneral : public QWidget {
 
     void SetupPerGameUI();
 
+    void (*ResetCallback)(ConfigureDialog*);
```

Comment:
```suggestion
    std::function<void(ConfigureDialog*)> reset_callback;
```

Ditto with `SetResetCallback()`, which would allow lambda functions with captures to work with the interface.

Perhaps also make an alias like:

```cpp
using ResetCallbackFn = std::function<void(ConfigureDialog*)>;
```

So the verbose-ish part only needs to be specified once.